instrumentation/opentelemetry-instrumentation-sqlalchemy/src/opentelemetry/instrumentation/sqlalchemy/engine.py

```diff
@@ -153,9 +186,8 @@ def _operation_name(self, db_name, statement):
             return self.vendor
         return " ".join(parts)
 
-    # pylint: disable=unused-argument
     def _before_cur_exec(
-        self, conn, cursor, statement, params, context, executemany
+        self, conn, cursor, statement, params, context, _executemany
```

Comment:
What is this change?

Reply:
This argument is not being used so figured out that I can use underscore instead of writing:
"# pylint: disable=unused-argument"
like,
https://github.com/open-telemetry/opentelemetry-python-contrib/blob/3bcc043e9dfc1b13421ff250fd941a40b306144d/instrumentation/opentelemetry-instrumentation-sqlalchemy/src/opentelemetry/instrumentation/sqlalchemy/engine.py#L199

Reply:
The disabled pylint always feels a bit hacky so I wanted to avoid this.
What do you think is better?

Reply:
Not a big opinion. I just prefer to not make changes if I don't need to.